### ISSUE TITLE
[Preview-MAS Violation]:During scan mode column heading are narrated twice.

### ISSUE THREAD

**Issue by NavyaGoberu** [nvaccess/nvda#9621]
User Experience:
User who rely on screen reader if when focus on the column header screen reader is narrating twice user will confuse.

Test Environment:
OS: Windows 10
Browser: Chrome
URL: https://int.onerf.microsoft.com/en-us/enterprise/tablev2?preview=inprogress&variationChoices=*:default

Repro Steps:
Step 1: Open the above URL using any browser.
Step 2: Navigate to section “OneGDCWeb-TableV2-yxfw15b” and press down key to read the table column header.
Step 3 : Verify while pressing down key column header information is narrate twice or not? 

Actual:
While pressing down key when when focus on the column header screen reader is narrating twice of the column header.
Expected:
Screen reader should be narrate the column header information single time.

MAS Reference:
MAS 4.1.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Comment by DrSooom** [comment]
@NavyaGoberu: Please specify the Win10, the Google Chrome and the NVDA version in the issue description. Thanks.

**Comment by Adriani90** [comment]
Duplicate of #10066
